app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -1,7 +1,7 @@
-module UserEncryptedEmailOverrides
+module UserEncryptedAttributeOverrides
```

Comment:
I like that phone and email are using this shared logic /  concern! Did you think about putting `email` and `phone` logic into separate `EncryptedPhone` and `EncryptedEmail` classes that include this concern? I am not sure if that would be better, but it's an idea that occurred to me as I read this.